### ISSUE TITLE
Default users do not seem to work

### ISSUE THREAD

**Issue by dsobral** [decidim/docker#103]
Hello,

I have used the docker compose:
```
git clone git@github.com:decidim/docker.git decidim-docker
cd decidim-docker
docker-compose up

```

It all works, and I can create a user etc...
But I tried to login using the default user/admin/system users, and they don't seem to work.
https://docs.decidim.org/en/v0.27/install/ 
[system@example.org](mailto:system@example.org) 
[admin@example.org](mailto:admin@example.org)
the /admin or /system "link" redirect to the same place...

I've tried to find the information in the database, but I cannot find anything.

How do I login to admin / system in the docker instance?
Thanks.

Kind regards,
Daniel

**Comment by pluginaria** [comment]
I've had same problem. I have found out that docker compose file contains image:

ghcr.io/decidim/decidim:latest

But as of today this 'latest' tag is 3 years old. Perhaps default passwords were different 3 years ago.

Docker hub version decidim/decidim:latest is recent though.
So you can change docker compose file to use Docker hub image or to use specific version (ghcr.io/decidim/decidim:0.28.0 as of today).

Hope this helps to anyone that stumbles into this problem.

**Comment by punkyard** [comment]
yes, I dont know if the app is any good, but the administration and docs around is a mess

here is the access for the admin from the login page:
http://localhost:3300 and click login  
admin@example.org  
decidim123456

here is the access for the super admin:
http://localhost:3300/system  
http://0.0.0.0:3000/system/admins/sign_in  
system@example.org  
decidim123456

I hope this helps
